components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -226,9 +226,9 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
 
     // Successfully stopped and headless: the prebuild is done, let's try to use it!
     if (!error && workspaceInstance.status.phase === 'stopped' && this.state.workspace?.type !== 'regular') {
-      const contextURL = ContextURL.parseToURL(this.state.workspace?.contextURL);
+      const contextURL = ContextURL.getNormalized(this.state.workspace, false);
```

Comment:
Shouldn't this actually "fall back to raw" when there is no normalizedContextURL? 🤔

Could also be simplified:

```ts
this.redirectTo('/' + (this.state.workspace?.context?.normalizedContextURL || this.state.workspace?.contextURL)
```

Reply:
I don't think so: In case the workspace was a prebuild, we now want to start a fresh workspace on it's context and leverage it (cmp. comment 2 lines above).